test/ShapeCrawler.Tests.Unit/ShapeCollectionTests.cs

```diff
@@ -360,6 +360,7 @@ public void AddPicture_should_not_duplicate_the_image_source_When_the_same_svg_i
 
         // Act
         shapes.AddPicture(svgImage);
+        Task.Delay(1100).Wait(); // See issue #883
```

Comment:
I think using this statement in Act-block turns this test into a non-functional test. I consider unit tests more as functional/component tests, and Act-block contains only code that the library's client uses.

Maybe we should start moving such tests into a separate test project like "ShapeCrawler.Stress(Load)Tests". The tests in this project should not be run in parallel and should be run multiple times.

I do not have a lot of experience in load testing, and therefore I'm open to ideas.

Reply:
No worries. I have removed them. 952140244bc4e23144eff556e69eed2da53f73ea